### PR TITLE
Change enumset by big_enum_set and fix GenericName for amv2

### PR DESCRIPTION
**Pull request by Aursen**

### CODE REVIEW

File: swf/src/types.rs [hunk 1]
```diff
@@ -417,7 +417,7 @@ pub enum BlendMode {
 /// [SWF19 pp.37-38 ClipActionRecord](https://www.adobe.com/content/dam/acom/en/devnet/pdf/swf-file-format-spec.pdf#page=37)
 #[derive(Debug, Clone, PartialEq)]
 pub struct ClipAction {
-    pub events: EnumSet<ClipEventFlag>,
+    pub events: BigEnumSet<ClipEventFlag>,
```
Comment (adrian17) [inline swf/src/types.rs:420]: Feels to me like this just takes extra memory for no benefit. (previously it was stored in 4 bytes just like in the SWF itself, now it's stored in `usize`)

Comment (Aursen) [reply swf/src/types.rs:420]: Yes sorry, I modified this lib to compile with rust 1.50 in another project

**Comment by Herschel** [comment]
The `BigEnumSet` change seems unrelated, could you put that into a different PR please? (We do need to do something here because of https://github.com/Lymia/enumset/pull/18 , but we should change to https://github.com/Lymia/enumset/pull/18 , I don't see a pressing reason for `BigEnumSet`.) Thank you!

File: swf/src/types.rs (hunk 1 above)
Comment (Aursen) [reply swf/src/types.rs:420]: enumset is not compatible

**Pull request closed without merging**